src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -213,6 +213,20 @@ protected function getDefaultValueString(Column $column)
         return $defaultValue;
     }
 
+    /**
+     * Return the parent class name, or null.
+     *
+     * @return string|null
+     */
+    protected function getParentClass()
```

Comment:
Should we add return type for New methods?

Reply:
done!